src/network/networkd-netdev-tunnel.c

```diff
@@ -202,6 +202,7 @@ static int netdev_ip6gre_fill_message_create(NetDev *netdev, Link *link, sd_netl
 
 static int netdev_vti_fill_message_create(NetDev *netdev, Link *link, sd_netlink_message *m) {
         Tunnel *t = VTI(netdev);
+        unsigned ikey, okey;
```

Comment:
shouldn't this be uint32_t? You use sd_netlink_message_append_u32() to add it to th enetlink msg...